docs/user-guide/creating-application/secrets.md

```diff
@@ -97,12 +97,49 @@ You can install `External Secrets Operator` using charts store:
 3. If you don't find any chart with this name i.e `external-secrets`, add chart repository using repository url  ` https://charts.external-secrets.io`. Please follow this [documentation](https://docs.devtron.ai/getting-started/global-configurations/chart-repo#add-chart-repository) for adding chart repository.
 4. Deploy the chart.
 
+#### AWS Secrets Manager
+
+To add secrets from AWS secret manager, navigate to `Secrets` of the application and follow the steps mentioned below :
+
+1. Create generic secret in the namespace using AWS access-key and secret-access-key.
+
+```bash
+echo -n 'aws-access-key' > ./access-key
+```
+
+```bash
+echo -n 'aws-secret-access-key' > ./secret-access-key
+```
+
+Replace NAMESPACE with the namespace name in the command and run:
+
+```bash
+kubectl create secret generic awssm-secret --from-file=./ access-key  \
+--from-file=./secret-access-key -n NAMESPACE
+```
+
+2. Click `Add Secret` to add a new secret.
+
+![](https://devtron-public-asset.s3.us-east-2.amazonaws.com/images/creating-application/secrets/add-secrets.jpg)
+
+3. Select `AWS Secret Manager` (External Secret Operator) from dropdown of `Data type`.
+
+![](https://devtron-public-asset.s3.us-east-2.amazonaws.com/images/creating-application/secrets/aws-secret-manager-dropdown.jpg)
+
+4. Configure secret
+![](https://devtron-public-asset.s3.us-east-2.amazonaws.com/images/creating-application/secrets/configure-aws-secret.jpg)
+
```

Comment:
add a few statements mentioning the corresponding key and property parameters in AWS Secret Store that you showed in the image below.

Reply:
done